model/k8s/registry.yaml

```diff
@@ -214,3 +214,25 @@ groups:
               value: 'local'
               brief: "A [local](https://v1-29.docs.kubernetes.io/docs/concepts/storage/volumes/#local) volume"
```

Comment:
nit:

it seems we use 1.30 version below, can we reference the same version consistently?